packages/sources/nomics-test/src/endpoint/filtered.ts

```diff
@@ -0,0 +1,94 @@
+import { RestTransport } from '@chainlink/external-adapter-framework/transports'
+import { AdapterEndpoint } from '@chainlink/external-adapter-framework/adapter'
+import { InputParameters } from '@chainlink/external-adapter-framework/validation'
+import { customSettings } from '../config'
+import { AdapterError } from '@chainlink/external-adapter-framework/validation/error'
+
+export const inputParameters: InputParameters = {
+  base: {
+    aliases: ['from', 'coin', 'id'],
+    required: true,
+    type: 'string',
+    description: 'The symbol of the currency to query',
+  },
+  exchanges: {
+    required: true,
+    type: 'string',
+    description: 'Comma delimited list of exchange names',
+  },
+}
+interface ResponseSchema {
+  currency: string
+  price: number
+}
+
+interface RequestParams {
+  base: string
+  exchange: string
+}
+
+interface ProviderRequestBody {
+  currency: string
+  key: string
+  exchanges: string
+}
+
+export type FilteredEndpointTypes = {
+  Request: {
+    Params: RequestParams
+  }
+  Response: {
+    Data: {
+      result: number
+    }
+    Result: number
+  }
+  CustomSettings: typeof customSettings
+  Provider: {
+    RequestBody: ProviderRequestBody
+    ResponseBody: ResponseSchema
+  }
+}
+
+const restEndpointTransport = new RestTransport<FilteredEndpointTypes>({
+  prepareRequest: (p, config) => {
+    const baseURL = config.API_ENDPOINT
```

Comment:
Seems like nomics supports batches on the base side, but not the quote. Same thing as the kaiko one, we should block this one.

Reply:
Can this be unblocked now? Can we make this EA send batches on the base, like the crypto endpoint?

Reply:
I don't really think `/prices/restricted` API endpoint is partially batchable. As a matter of fact i didn't even find any documentation for that. There is a `prices` API endpoint where just one ticker is allowed.

Reply:
Should we just mark this as resolved then?